packages/components/src/box-control/unit-control.tsx

```diff
@@ -32,7 +32,7 @@ export default function BoxUnitControl( {
 
 	return (
 		<UnitControlWrapper { ...bindHoverGesture() }>
-			<Tooltip text={ label }>
+			<Tooltip placement="top" text={ label }>
```

Comment:
I replaced the Tooltip as it was just creating another version of the component with no differences. The `div` isn't needed anymore, but if it was, it could be added directly around the `UnitControl` anyway.